### PR TITLE
Add Slack Node.js Community

### PR DESCRIPTION
**Pull request by aymen94**
Added Slack Node.js Community

### CODE REVIEW

**Comment by ljharb** [comment]
cc @trott

**Comment by Trott** [comment]
So, the reason Slack is not on there is because it's not an official affiliated-with-the-Foundation chat like the other two were. Gitter, though, was shut down and should be removed.

I think the Node.js Slack adopted our Code of Conduct and there may be an argument to include it, although I don't know if it should be listed separately as unofficial like it is in the core repo. Let's /ping @nodejs/community-committee for that.

Anyway, this change is fine by me.

**Comment by ljharb** [comment]
@Trott it did adopt it; myself and other members of the moderation team are some of the admins of it.

**Comment by ljharb** [comment]
As for the gitter, it's just as much as in use as it's been for years - i think you mean it's discord that was shut down.

**Comment by Trott** [comment]
> As for the gitter, it's just as much as in use as it's been for years - i think you mean it's discord that was shut down.

No, I'm thinking of this: https://github.com/nodejs/node/pull/13231 It wasn't the lack of use that got it removed as much as the lack of moderation, I **think**... See also https://github.com/nodejs/node/pull/13502#issuecomment-306554977. Anyway, all that was from 2017 and the Community Committee was just getting started. I'm happy to defer to them on guidelines for this stuff now. I do think we would be wise to choose only a very small number of durable high-quality resources and resist adding lots and lots of things. It's better for the user if there are only two things that are great and not two things that are great but they get lost in a list of 90 things that are hit-and-miss.

**Comment by Trott** [comment]
(You're right, though, that the gitter was not shut down, obviously. I got that wrong for sure!)

**Comment by Bamieh** [comment]
We use it as our main communication channel for mentorship. we also have a community committee channel there. I'd encourage having it on the list even if we label it * *unofficial* *

**Comment by WaleedAshraf** [comment]
We have it in moderation: https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#admins-for-nodejs-slack-community

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ this repository, you can try:
 
 * [Freenode #node.js channel](https://webchat.freenode.net/?channels=node.js&uio=d4)
 * [Gitter Node.js room](https://gitter.im/nodejs/node)
```
Comment (Trott) [inline README.md]: Can you remove the Gitter line while we're in here?

Comment (aymen94) [reply README.md]: Done! 😄

**Comment by Chasdaspaz** [comment]
pull request

**Comment by anhthiet92buh** [comment]
Help me, please! How to fix thix?
image

**Pull request merged**